docs/contributing/app-dev.md

```diff
@@ -42,15 +42,15 @@ If you need to run the application on a different port than the default (8000),
 
 ```bash
 export STREAMLIT_SERVER_PORT=1234
-streamlit run app.py
+streamlit run src/app.py
 ```
 
 ## Project Layout
 
 ### Application files
 
-- `app.py`: Main source for the application
-- `test_app.py`: [pytest](https://docs.pytest.org/en/latest/) tests for `app.py`
+- `penn_chime/app.py`: Main source for the application
+- `penn_chime/test_app.py`: [pytest](https://docs.pytest.org/en/latest/) tests for `app.py`
```

Comment:
@quinn-dougherty looks like we missed a spot

Reply:
thanks